runner/package.json

```diff
@@ -38,6 +39,7 @@
     "@endo/init": "^1.1.8",
     "@endo/promise-kit": "^1.1.9",
     "@iarna/toml": "^2.2.3",
+    "agoric": "dev",
```

Comment:
Why is this a dependency? The whole point was to remove it.

Reply:
That'd be grand! Is there a PR that removes it?

The latest I've seen still needs `getSDKBinaries`:
https://github.com/Agoric/testnet-load-generator/blob/9cc503b7357038ff1a3b8115d55dc232e1f5c919/runner/lib/main.js#L163-L171

Reply:
I thought I wrote a fallback to `SDK_SRC` somewhere. I'll dig it out.

Reply:
https://github.com/Agoric/testnet-load-generator/compare/483daf90fdd1febf0bfa8f885474e56792c12885...standalone-runner

I'm surprised those are not part of #119. I need to review that PR.